infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/KubernetesInternalRuntime.java

```diff
@@ -212,6 +211,14 @@ protected void internalStart(Map<String, String> startOptions) throws Infrastruc
       final EnvironmentContext currentContext = EnvironmentContext.getCurrent();
       CompletableFuture<Void> startFailure = startSynchronizer.getStartFailure();
 
+      final Scope waitRunningAsyncSpan =
```

Comment:
Could you move this to a separate method? This would simplify the code.
```

Reply:
Moved to `TracerUtil`

Reply:
Moved to `TracerUtil`.